app/frontend/src/features/profile/view/leaveReference/formSteps/submit/SubmitFriendReference.tsx

```diff
@@ -0,0 +1,75 @@
+import { useMediaQuery, useTheme } from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import { REFERENCE_SUCCESS, SUBMIT } from "features/profile/constants";
+import { useWriteFriendReference } from "features/profile/hooks/referencesHooks";
+import ReferenceOverview from "features/profile/view/leaveReference/formSteps/submit/ReferenceOverview";
+import { useReferenceData } from "features/profile/view/leaveReference/ReferenceDataContext";
+import {
+  ReferenceFormProps,
+  useReferenceStyles,
+} from "features/profile/view/leaveReference/ReferenceForm";
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { WriteFriendReferenceInput } from "service/references";
+
+export default function SubmitFriendReference({ user }: ReferenceFormProps) {
+  const theme = useTheme();
+  const classes = useReferenceStyles();
+  const isMdOrWider = useMediaQuery(theme.breakpoints.up("md"));
+  const {
+    writeFriendReference,
+    status: writingStatus,
+    reset: resetWriting,
```

Comment:
you only have one mutation, I would just leave it as `status` and `reset`. Or `mutationStatus` and `resetMutation`